### PR TITLE
Many fixes in various Mads modules

### PR DESCRIPTION
**Pull request by montyvesselinov**

### CODE REVIEW

**Comment by tkelman** [comment]
Anasol is missing DocumentFunction from its REQUIRE

**Comment by montyvesselinov** [comment]
oops. v0.2.5 of Anasol is pointing now to a new tag.

-monty

**Comment by tkelman** [comment]
can you rebase that commit out from here? looks like attobot opened a separate pr for it

**Comment by montyvesselinov** [comment]
this should work now

**Comment by montyvesselinov** [comment]
Travis is successful.

File: Mads/versions/0.4.12/requires [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Measures
+Images
+YAML
+JSON
+JLD
+ProgressMeter
+Distributions
+DataStructures
+DataFrames
+StatsBase
```
Comment (tkelman) [inline Mads/versions/0.4.12/requires]: `StatsBase.Weights` requires 0.15

**Comment by montyvesselinov** [comment]
Now StatsBase is fixed

**Pull request merged**